blog/_posts/2023-01-24-scala-center-2023-roadmap.md

```diff
@@ -0,0 +1,247 @@
+of implementing the `debugSession/start` endpoint in all the build servers by running the debug server in Metals,
+update scalameta to Scala 3, publishing sbt 2, implement a Scala 2 expression evaluator in scala-debug-adapter,
+be able to choose a specific build target in Metals, make sure Scala programs are easy to deploy especially in IaaS,
+and provide TASTy trees to semantic rules in Scalafix.
+
+### Community and Contributor Experience
+
+In addition to continuing co-organizing ScalaDays, participating to conferences, publishing positive content about
+Scala, leading the Compiler Academy, and maintaining tools for contributors ([Scastie](https://scastie.scala-lang.org),
+[Scaladex](https://index.scala-lang.org), [scalafix](https://scalacenter.github.io/scalafix/)), we will:
+
+- **Add support for standalone Scala programs in Scastie.** Currently, Scastie creates an sbt project for every snippet.
+  This adds a performance overhead, and makes it impossible to just copy-paste a snippet to run it locally (because
+  you also need to re-create the sbt configuration locally). We will fix this issue by supporting a “standalone” mode
+  where the configuration will be defined via Scala CLI directives.
+- **Publish TASTy-MiMa.** This tool will be able to detect incompatibilities between two versions of a program, based
+  on their TASTy representation. We will complete its implementation and make it available to the community.
+- **Streamline the way to publish the documentation of Scala libraries.** Currently, there are many tools one can use
+  to publish documentation about their open-source projects (Scaladoc, Paradox, Laika, Docusaurus, etc.). Developers
+  should not have to study or experiment with many tools to produce the documentation they need. We will simplify how
+  to produce API documentation, how to “typecheck” the code examples, how to keep multiple versions of the documentation
+  online, and how to define custom “template directives” in Scala.
+
+Our stretch goals include: coordinate Google Summer of Code, create a Scala Center online shop, stabilize and document
+the HTTP API of Scaladex, make it easy to find projects to contribute to in Scaladex, implement GitHub [precise code
+navigation](https://docs.github.com/en/repositories/working-with-files/using-files/navigating-code-on-github#precise-and-search-based-navigation)
+for Scala, and simplify the usage of remote caching techniques in continuous integration systems.
+
+## Conclusion
+
+In this article, we have looked at the pillars of the Scala ecosystem, and for each of them we have listed the main
+outcome of our work in 2022, and our main goals for 2023.
+
+Achieving these goals is possible only with your [support](https://scala.epfl.ch/faqs.html) and with the help of all
+the [people behind Scala](https://www.scala-lang.org/community/#whos-behind-scala). Thank you all!
```

Comment:
I'd suggest inserting a little fundraising verbiage here, encouraging companies to join, and/or encouraging employees to encourage their employers to join. The "support" link is there, of course, but I think it would be appropriate and desirable to push it a little harder.

Reply:
I’ve reworked this part in 1ac8cfb4.